### DISCUSSION TITLE
Use a design system as source

### DISCUSSION THREAD

**Discussion by sjur-gr** [webstudio-is/webstudio-community#36] [Q&A]
Yo!

I have a design system with design tokens and react components. (layout components and UI components). Would l be able to integrate my design system and use the tokens and components in Webstudio?

**Comment by ghost** [comment]
That is absolutely our intention and your question represents a primary
use-case for Webstudio. However it’s still early, so “tokens sync” and
“user components” are features still in development.

Stay tuned, those features are definitely coming to Webstudio in the near
future!
